src/main/java/uk/gov/ch/oauth/IdentityProvider.java

```diff
@@ -95,6 +95,16 @@ public String getAuthorisationUrl(final String state) {
         return url;
     }
 
+    @Override
+    public String getPostRequestBody(String code) {
+        final StringBuilder sb = new StringBuilder();
+        sb.append("code=").append(code).append("&client_id=").append(getClientId())
+                .append("&client_secret=").append(getClientSecret()).append("&redirect_uri=")
+                .append(getRedirectUri()).append("&grant_type=authorization_code");
```

Comment:
Should authorisation_code be a constant?
Would this be more readable as
`return String.format("code={}&client_id={}&client_secret={}&redirect_uri={}&grant_type=authorization_code",
     code,
     getClientId(),
    getClientSecret(),
    getRedirectUri();
`
Except maybe with authorization code also being a constant